packages/@uppy/audio/src/PermissionsScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import type { I18n } from '@uppy/utils/lib/Translator'
-import { h } from 'preact'
+import type { h } from 'preact'
```

Comment:
should not be a type

Reply:
why?

> I've tested `yarn dev` and it works without `import { h} from 'preact'`. same with examples

also there's no build errors or tests failing